tripy/tests/test_examples.py

```diff
@@ -80,12 +80,43 @@ def __str__(self):
         return os.path.relpath(self.path, EXAMPLES_ROOT)
 
 
-EXAMPLES = [Example(["nanogpt"])]
+EXAMPLES = [Example(["nanogpt"]), Example(["segment-anything-model-v2"])]
 
 
 @pytest.mark.l1
 @pytest.mark.parametrize("example", EXAMPLES, ids=lambda case: str(case))
 def test_examples(example, sandboxed_install_run):
+
+    def test_with_tolerance(number, value, tolerance):
```

Comment:
nit: Can we rename to `actual`/`expected`? Unclear which of `number`/`value` is the ground truth otherwise